modules/client-configuration/pages/clients-ubuntu.adoc

```diff
@@ -144,6 +144,13 @@ The root user on {ubuntu} is disabled by default for SSH access.
 
 To be able to onboard using a regular user, you need to edit the [filename]``sudoers`` file.
 
+[NOTE]
+====
+This issue happens with self-installed versions of {ubuntu}.
+If the default user has been granted administrative privileges during installation time, a password is requiered to perform privilege escalation using [command]``sudo``.
+With cloud instances this does not happen since [command]``cloud-init`` automatically creates a file under [path]``/etc/sudoers.d`` and grants privilege escalation through [command]``sudo`` without the need for a password.
```

Comment:
```suggestion
With cloud instances this does not happen because [command]``cloud-init`` automatically creates a file under [path]``/etc/sudoers.d`` and grants privilege escalation through [command]``sudo`` without the need for a password.
```
We try to avoid "since" when because can be used.

Reply:
Wording changed @keichwa !